macaw-aarch32/tests/ARMTests.hs

```diff
@@ -113,17 +127,32 @@ getExpected expectedFilename = do
           ignoredBlocks = S.fromList (ignoreBlocks er)
       in return (expectedEntries, ignoredBlocks)
 
+escapeSlash :: Char -> Char
+escapeSlash '/' = '_'
+escapeSlash c = c
+
+toSavedMacawPath :: String -> FilePath
+toSavedMacawPath testName = "/tmp" </> name <.> "macaw"
```

Comment:
I'd recommend simply writing to the current directory.  If this flag was ever enabled for CI I don't know that `/tmp` would necessarily be available, or if it was, there's a risk of the wrong output due to parallel tests.  You could also use the contents of a `TEST_OUTDIR` env var lookup with fallback to `.` if you wanted flexibility.